ecr-login/cache/file.go

```diff
@@ -98,6 +98,9 @@ func (f *fileCredentialCache) fullFilePath() string {
 // file access. There is not guarantee here for handling multiple writes at once since there is no out of process locking.
 func (f *fileCredentialCache) save(registryCache *RegistryCache) error {
 	defer log.Flush()
+	if _, err := os.Stat(f.path); err != nil {
```

Comment:
Are we okay with having the log and the cache file stored in different directories? The logs gets initialized before the constructor, so if this constructor is used, the log and the cache file will end up in different directories.

If customers are using both the credential helper and the cli, without it make sense for us to store and use the cache file from the same directory?